packages/typespec-client-generator-core/test/package.test.ts

```diff
@@ -349,7 +349,7 @@ describe("typespec-client-generator-core: package", () => {
       const method = getServiceMethodOfClient(sdkPackage);
       strictEqual(method.name, "create");
       strictEqual(method.kind, "basic");
-      strictEqual(method.parameters.length, 5);
+      strictEqual(method.parameters.length, 4);
```

Comment:
why just length updated? but below deepStrictEqual for param name is not updated?

Reply:
fixed